Resources migrated and excluded by MTC (#29096)
openshift/openshift-docs: modules/migration-mtc-workflow.adoc

[id='migration-mtc-workflow_{context}']
= {mtc-full} workflow

> The {mtc-full} ({mtc-short}) enables you to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster, using the {mtc-short} web console or the Kubernetes API.

Reviewer: ```suggestion
You use the {mtc-full} ({mtc-short}) to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster by using the {mtc-short} web console or the Kubernetes API.
```

Reviewer: @apinnick, what do you think of the suggested edit?

Reviewer 2: Fine

The ({mtc-short}) migrates the following resources:

* A namespace specified in a migration plan.

* Namespace-scoped resources: When the {mtc-short} migrates a namespace, it migrates all the objects and resources associated with that namespace, such as services or pods. Additionally, if a resource that exists in the namespace but not at the cluster level depends on a resource that exists at the cluster level, the {mtc-short} migrates both resources.
+

> For example, a security context constraint (SCC) is a resource that exists at the cluster level and a service account (SA) is a resource that exists at the namespace level. If an SA exists in a namespace that the {mtc-short} migrates, the {mtc-short} automatically locates any SCCs that are linked to the SA and also migrates those SCCs. Similarly, the {mtc-short} migrates persistent volume claims linked to the persistent volumes of the namespace.

Reviewer: ```suggestion
For example, a security context constraint (SCC) is a resource that exists at the cluster level and a service account (SA) is a resource that exists at the namespace level. If an SA exists in a namespace that the {mtc-short} migrates, the {mtc-short} automatically locates any SCCs that are linked to the SA and also migrates those SCCs. Similarly, the {mtc-short} migrates persistent volume claims that are linked to the persistent volumes of the namespace.
```

> * Custom resources (CRs) and custom resource definitions (CRDs): The {mtc-short} automatically migrates any CRs that exist at the namespace level as well as the CRDs linked to those CRs.

Reviewer: ```suggestion
* Custom resources (CRs) and custom resource definitions (CRDs): The {mtc-short} automatically migrates any CRs that exist at the namespace level as well as the CRDs that are linked to those CRs.
```
